compute/admin_guide/authentication/oidc.adoc

```diff
@@ -164,7 +164,15 @@ You can only view the value of the secret when you create it. Copy and store the
 [.procedure]
 . To add the needed claim, go to *Token Configuration*.
 
-. Add a group claim and select the *Groups assigned to the application* option.
+. Add *Optional claims* of Token type - ID. Check mark *email* and *preferred_username* claims. While saving these claims, it will prompt to enable Microsoft Graph email permission, enable it. Azure OIDC application should provides these claims to the Prisma Cloud Console to authenticate and render user info.
+
+image::oidc_optional_claim.png[width=250]
+
+. Add Microsoft Graph - Delegated - API permissions for *email, openid, profile*. Azure OIDC Application requires these permissions for accessing the API as the signed-in user.
+
+image::oidc_api_permission.png[width=250]
+
+. Add above group claim and select the *Groups assigned to the application* option.
```

Comment:
@priteshchandaliya The steps above do not mention any groups, they are mentioned below. Do we need to add this step or should the step that follows needs to be modified?